jobs/github-event/installation_repositories/added.js

```diff
@@ -10,31 +10,31 @@ const { createDocs } = require('../../../lib/repository-docs')
 
 const max404Retries = 5
 
-module.exports = async function ({ installation, repositories_added }) {
+module.exports = async function ({ installation, repositoriesAdded }) {
```

Comment:
in the github event that is incoming it sends a data object and in it is `repositories_added`  so renaming it here would result in no data. But we could do `{ installation, repositoriesAdded: repositories_added }` I think?

Reply:
okay!